lib/unicorn_wrangler.rb

```diff
@@ -41,15 +42,24 @@ def setup(
         end
 
         @hooks[:after_fork] = ->(*) do
-          Signal.trap :TERM do
-            Thread.new { logger.info 'worker intercepting TERM and doing nothing. Wait for master to send QUIT' }
+          old = Signal.trap :TERM do
```

Comment:
May want to add a comment here explaining what the hell is going on here, and that yes, you can use `old` in the block you pass in here.